invenio/templates/deployments/web.yaml

```diff
@@ -90,6 +90,9 @@ spec:
               key: {{ default (printf "INVENIO_%s_APP_CREDENTIALS" .name) }}
         {{- end }}
         {{- end }}
+        {{- if .Values.extraEnv }}
```

Comment:
I am wondering if instead of putting a *catch all* config here, we could put it in the `configurations/invenio.yaml`. Unless we are talking about secrets? Note that even if not an `INVENIO_` they would go to the config file (we do that for e.g. `FLASK_` variables.

Reply:
I thought of that solution, but in the end I opted for a way to do all configuration in just one place (the values.yaml). Just don't like the idea I have to change files from the chart itself and maintain all my changes through all next updates.

Reply:
Sorry, I think I did not explain myself 😅. I do agree with config coming all the way from one place (values.yaml). My point was about *where to do the expansion*. Right now it is being don in `web.yaml`, `worker.yaml`, etc. which leads to repeated code and if needed to be changed without helm: editing a yaml file manually in the deployement. What I was suggesting was that we would have only:

`config.yaml`
```
        {{- if .Values.extraEnv }}
         {{- toYaml .Values.extraEnv | nindent 8 }}
         {{- end }}
```

WDYT?

Reply:
Great idea :+1: That makes it just more easy to maintain the chart! I'm in complete agreement!